clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -2165,7 +2164,12 @@ def TLSModel : I32EnumAttr<
     [TLS_GeneralDynamic, TLS_LocalDynamic, TLS_InitialExec, TLS_LocalExec]> {
   let cppNamespace = "::mlir::cir";
 }
-  
+
+
+def AnnotationArrayAttr :
```

Comment:
This doesn't feel like a one shot thing, should be in CIRAttrs.td and declared on top of CIR_Attr

Reply:
Call it `GlobalAnnotationAttr` and have it's input be a `ArrayAttr`.